theme/themes/pastanaga/elements/step.variables

```diff
@@ -95,7 +95,7 @@
 
 @attachedHorizontalOffset: -@borderWidth;
 @attachedVerticalOffset: 0;
-@attachedWidth: ~"calc(100% + "-@attachedHorizontalOffset * 2~")";
+@attachedWidth: calc(100% + -@attachedHorizontalOffset * 2);
```

Comment:
maybe you should keep the `~" "` around https://github.com/less/less.js/issues/974#issuecomment-9229470